.github/pull-request-template.md

```diff
@@ -0,0 +1,57 @@
+## Related Issue (if applicable)
+Fixes #{issue number}
+
+## Summary
+What are you trying to achieve in this PR? Please summarize what changes you made and how they acheive the desired result.
+
+## Changes checklist (if adding new augmentation)
+If you are adding a new augmentation, please ensure that you have made the necessary changes in all of the following files, where `<module>` is the module (`audio`, `text`, or `image`) where you added the new augmentation.
+- `augly/<module>/functional.py`
+- `augly/<module>/transforms.py`
+- `augly/<module>/intensity.py`
+- `augly/<module>/__init__.py`
+- `augly/tests/<module>_tests/functional_unit_tests.py`
+- `augly/tests/<module>_tests/transforms_unit_tests.py`
+- `augly/utils/expected_output/<module>/expected_metadata.json`
+- `augly/assets/tests/<module>_tests/test_<aug_name>.py`
+
+Note about test files: The final file will be added, and should be the result of running the new augmentation with the args as specified in the new unit tests. The two new unit tests in `functional_unit_tests.py` & `transforms_unit_tests.py` should specify the same arguments for the new augmentation so they can use the same output file.
+
+Note about new video augmentations: You may also need to add an augmenter file in `augly/video/augmenters/` if your augmentation uses `cv2` or `ffmpeg`. Instead of `functional_unit_tests.py` or `transforms_unit_tests`, you should choose which test file to add the unit test for your new augmentation to based on which category it best fits in to. If the augmentation is implemented using `cv2` or `ffmpeg`, you should add the unit test to `cv2_tests.py` or `ffmpeg_tests.py` respectively. If the augmentation is implemented by applying a function to each frame of the video, you should add the unit test to `image_based_tests.py`. If the augmentation does some kind of overlay or spatial distortion, you should add the unit test to `composite_tests.py`.
+
+Please implement your new augmentation using AugLy's existing dependencies (i.e. `PIL`, `numpy`, `nlpaug`, `librosa`, etc.) if possible, and avoid adding new dependencies as these will make AugLy heavier and slower to download. However, if you feel it's necessary in order to implement your new augmentation and that the new augmentation is really worth having, it may be fine; in this case, add your new dependency to `requirements.txt`, and then make sure you can install `augly` in a fresh conda environment and the new unit test passes.
```

Comment:
```suggestion
Please implement your new augmentation using AugLy's existing dependencies (i.e. `PIL`, `numpy`, `nlpaug`, `librosa`, etc.) if possible, and avoid adding new dependencies as these will make AugLy heavier and slower to download. However, if you feel it's necessary in order to implement your new augmentation and that the new augmentation is really worth having, it may be fine; in this case, add your new dependency to `requirements.txt`, and then make sure you can install `augly` in a fresh conda environment and the new unit tests pass.
```